src/task/SubscribeAndReceiveSubscribeAckTask.ts

```diff
@@ -107,6 +108,12 @@ export default class SubscribeAndReceiveSubscribeAckTask extends BaseTask {
         }
 
         handleSignalingClientEvent(event: SignalingClientEvent): void {
+          if (event.isConnectionTerminated()) {
```

Comment:
Do we retry when we are not mid-subscribe?  what about when we are mid Join-JoinAck?

Reply:
We retry when we are in mid-subscribe when WebSocket connection is terminated. An error will be thrown and meeting session status code will be taskFailed. It will be handled with retry [here](https://github.com/aws/amazon-chime-sdk-js/blob/master/src/audiovideocontroller/DefaultAudioVideoController.ts#L1069). The in join-joinAck task, the signaling client event will be handled here: https://github.com/aws/amazon-chime-sdk-js/blob/master/src/task/JoinAndReceiveIndexTask.ts#L49 It will retry for signaling internal server errors.

Reply:
Thanks for looking into this!